src/Common/perf/PerfRunner/project.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0.0-*",
```

Comment:
No this should not be used by anyone else.  I will take a look at how the RemoteExecutorConsoleApp build works and model this after that.